contrib/python-hatch-fancy-pypi-readme/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "python-hatch-fancy-pypi-readme"
+pkgver = "24.1.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-hatchling",
+    "python-installer",
+    "python-wheel",
+]
+depends = [
+    "python",
```

Comment:
don't need python also when another python dep is there, same for hostmakedepends

Reply:
mhm, oki